src/Intl/Idn/bootstrap.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Polyfill\Intl\Idn as p;
+
+if (!function_exists('idn_to_ascii')) {
+    define('IDNA_DEFAULT', 0);
+    define('INTL_IDNA_VARIANT_2003', 0);
+    define('INTL_IDNA_VARIANT_UTS46', 1);
+
+    if (\PHP_VERSION_ID < 70400) {
+        define('INTL_IDNA_VARIANT_DEFAULT', INTL_IDNA_VARIANT_2003);
```

Comment:
is this a constant defined in Intl ? If no, we should maybe define a namespaced constant to keep it in the polyfill)

Reply:
No this is a custom constant
What do you mean by namespaced constant ? Something like 
`define('Symfony\Polyfill\Tests\Intl\Idn\VARIANT_DEFAULT', ...);` ?

Reply:
yes, so that it is clear that this is not part of the polyfilled API